cpp/tensorrt_llm/kernels/quantization.cuh

```diff
@@ -794,6 +790,10 @@ quantize_with_block_size(
     int numColThreadsForSf = numColsForSf / ELTS_PER_THREAD;
 
     asm volatile("griddepcontrol.wait;");
+    // Get the global scaling factor, which will be applied to the SF.
+    // Note SFScale is the same as next GEMM's alpha, which is (448.f / (Alpha_A / 6.f)).
+    float const SFScaleVal = SFScale == nullptr ? 1.0f : SFScale[0];
```

Comment:
Global scale (for nvfp4) is supposed to be a constant, so it should be fine to load it before ACQBULK. mxfp4/mxfp8 do not have global scale